interview-questions.md

```diff
@@ -7,44 +7,93 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: I would begin by definining what a foreign key is which is known a column that connects two data keys in a table of data within a relaitonal database. In other words, a foriegn key acts almost like a holding of data that references a primary key in  the orginal table. I would fix this problem by  simply adding the foreign key by updating the attributes within a column. 
 
   Researched answer:
+I can do what I mentioned previously about adding a new column but I would identify this column by adding and integer to the cohort_id:integer
+I can rails db:migrate in rails app cohort.rb
+go to has_many students  and then add student model cohoty_id: integer
 
+I can also restart everything and add my foriegn key fromt he start but that would be a waste time. Migrating would be best.
```

Comment:
I agree!  When working with well established applications, restarting is not an option, so migration is the way to go!